test/edu/csus/ecs/pc2/core/model/SampleContest.java

```diff
@@ -2054,4 +2057,12 @@ public IInternalContest loadSampleContest(IInternalContest contest, String sampl
         }
     }
     
+    public void ensureStaticLog() {
+        
+        if (StaticLog.getLog() == null) {
+            String testDirectory = "logs";
+            StaticLog.setLog(new Log(testDirectory, "testPrintreport.log"));
```

Comment:
The name for the new Log file doesn't make sense and is misleading; this change doesn't seem to have anything to do with "printing a report".  I'm not sure what it SHOULD be -- maybe "testLoadSampleContest.log"?

Reply:
Note that all of these are out of scope.

Reply:
Wait a minute:  "out of scope"?   I could _easily_ argue that the entire process of adding a method to "ensureStaticLog()" is "out of scope" for a PR which is supposed to be supporting load_accounts.  The fact that you chose to put such a method in the PR means to me that you deemed it "in scope".   In return, I deem that it has been done improperly, leading to increasing the need to do "someday" fixes -- which nobody ever gets to.  I think when code is added to the system and somebody points out a reasonable flaw in the structure of the code (such as an inappropriately duplicated method), that flaw should be fixed when it's found rather than just linger and increase the flaws in the system -- because if it's not fixed now, when it's identified and it's clear in the minds of both the developer and the reviewer, when WILL it get fixed?

Reply:
Move ensureStaticLog() method into LogUtilities
Log name is now: LogUtilities.log based on class name where log was created
Note a refactoring of our logging is long overdue

Reply:
pushed fix.